enos/enos-scenario-autopilot.hcl

```diff
@@ -35,7 +36,8 @@ scenario "autopilot" {
       amd64 = "t3a.small"
       arm64 = "t4g.small"
     }
-    vault_instance_type = coalesce(var.vault_instance_type, local.vault_instance_types[matrix.arch])
+    vault_instance_type   = coalesce(var.vault_instance_type, local.vault_instance_types[matrix.arch])
+    set_undo_logs_env_var = semverconstraint(var.vault_product_version, ">=1.12.0-0") && semverconstraint(var.vault_product_version, "<1.13.0-0") ? true : false
```

Comment:
can name the local variable different than the `step`, may be call it `enable_undo_logs` ?